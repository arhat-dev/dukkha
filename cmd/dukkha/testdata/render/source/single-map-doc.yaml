foo@env: GLOBAL_ENV_FOO=${GLOBAL_ENV_FOO}
bar@shell: template:getDebianArch \"amd64\"
woo@template:
- hello {{ getAlpineArch "arm64" }}
