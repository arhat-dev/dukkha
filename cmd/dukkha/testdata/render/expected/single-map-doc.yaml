bar: amd64
foo: GLOBAL_ENV_FOO=foo
woo:
  - hello aarch64
